docs/tutorials/add-a-pallet/use-a-pallet.md

```diff
@@ -6,7 +6,7 @@ Now you are ready to compile and run your node that has been enhanced with nickn
 from the Nicks pallet. Compile the node in release mode with:
 
 ```bash
-cargo build --release
+WASM_BUILD_TOOLCHAIN=nightly-2020-10-05 cargo build --release
```

Comment:
All these tutorials use 2.0.0?

You should use a nightly that is around the release of 2.0.0.

And we should also mention which version should be installed for the tutorials.

Reply:
Yes, all our tutorials are on v2.0.0 :tada: Furthermore, each tutorial ([including this one](https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/blob/source/docs/tutorials/add-a-pallet/index.md#install-the-node-template)) mentions the specific version of Substrate/the Node Template on which it is built :heavy_check_mark:

Can you recommend a specific nightly? I was under the impression that 2020-10-05 was "the right" one to use w/ v2.0.0. Is this incorrect? Is there a compelling reason to move to another nightly? I have been specifying 2020-10-05 for a while now and I would prefer not to change things if it's not necessary.

Reply:
If that works it is okay, but there was some other nightly that broke the compilation in between as well and I think it was before october.

Reply:
I believe we have constantly been referencing 2020-10-05 with success and no one reporting issues afaik @danforbes @bkchr